common/start-performance.sh

```diff
@@ -239,6 +244,11 @@ else
     exit 1
 fi
 
+# Enable high concurrency mode if the concurrency type contains 1000 or more
+if [[ $concurrency_type =~ ^([0-9]{4}-[0-9]{3}|[0-9]{3}-[0-9]{4}|[0-9]{4}-[0-9]{4})$ ]]; then
```

Comment:
what would be the value for concurrency_type?

Reply:
Its the value assigned from jenkins from. Its in the format xxx-xxxx

Reply:
Renamed vairable with https://github.com/wso2/performance-is/pull/201/commits/66337b94469e2592335f0fb4006584ae7e581ba8